src/index.html

```diff
@@ -5,8 +5,460 @@
+            class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="product" data-qa="card">
+          <img
+          class="product__photo"
+          src="./images/imac.jpeg"
+          alt="APPLE iMac photo"
+          >
+          <h2 class="product__title-text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product__code-text">
+            Product code: 195434
+          </p>
+
+          <div class="product__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="product__reviews-text">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="product__price">
+            <p class="product__price-text">
+              Price:
+            </p>
+
+            <p class="product__price-value">
+              $2,199
+            </p>
+          </div>
+
+          <a
+          class="product__button"
+          href="index.html"
+          data-qa="card-hover"
+          >
+            BUY
+          </a>
+
+        </div>
+
+        <div class="product">
+          <img
+          class="product__photo"
+          src="./images/imac.jpeg"
+          alt="APPLE iMac photo"
+          >
+          <h2 class="product__title-text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product__code-text">
+            Product code: 195434
+          </p>
+
+          <div class="product__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="product__reviews-text">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="product__price">
+            <p class="product__price-text">
+              Price:
+            </p>
+
+            <p class="product__price-value">
+              $2,199
+            </p>
+          </div>
+
+          <a
+          class="product__button"
+          href="index.html">
+            BUY
+          </a>
+
+        </div>
+
+        <div class="product">
+          <img
+          class="product__photo"
+          src="./images/imac.jpeg"
+          alt="APPLE iMac photo"
+          >
+          <h2 class="product__title-text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product__code-text">
```

Comment:
Add a blank line between the blocks
```suggestion
          >
          
          <h2 class="product__title-text">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h2>
          
          <p class="product__code-text">
```